lib/vsc/install/shared_setup.py

```diff
@@ -331,7 +332,15 @@ def __init__(self, *args, **kwargs):
                 kwargs['output'] = xmlrunner_output
                 xmlrunner.XMLTestRunner.__init__(self, *args, **kwargs)
 
-        main_orig = unittest.main
+        # old setuptools
+        main_name = 'main'
```

Comment:
in older setuptools versions run_tests uses `unittest.main`, not `main`, so it's clearer to monkeypatch `setuptools.command.test.unittest.main`?